tests/lib/config/config-initializer.js

```diff
@@ -232,25 +232,25 @@ describe("configInitializer", function() {
                     format: "JSON",
                     commonjs: false
                 };
-                origLog = console.log;
-                console.log = function() {}; // necessary to replace, because of progress bar
+
+                sandbox = sinon.sandbox.create();
+                sandbox.stub(console, "log"); // necessary to replace, because of progress bar
+
                 process.chdir(fixtureDir);
+
                 try {
                     config = init.processAnswers(answers);
                     process.chdir(originalDir);
                 } catch (err) {
 
-                    // if processAnswers crashes, we need to be sure to restore cwd and console.log
-                    console.log = origLog;
+                    // if processAnswers crashes, we need to be sure to restore cwd
                     process.chdir(originalDir);
                     throw err;
+                } finally {
+                    sandbox.restore();  // restore console.log()
                 }
             });
 
-            beforeEach(function() {
-                console.log = origLog;
-            });
```

Comment:
I noticed that this beforeEach was trying to undo the before method's stubbing of console.log. That could also just be done in the before method itself. I do so as a try/catch/finally to ensure it is restored in happy and error paths. I believe this makes the logic around console.log stubbing more localized, and therefore more readable. If anyone disagrees, please let me know how I should do this instead.